fsh-generated/resources/StructureDefinition-Organization.json

```diff
@@ -64,7 +43,67 @@
             ]
           }
         ],
-        "mustSupport": true
+        "mustSupport": true,
+        "_short": {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/translation",
+              "extension": [
+                {
+                  "url": "lang",
+                  "valueCode": "de-DE"
+                },
+                {
+                  "url": "content",
+                  "valueString": "Beschreibung der Sammlung/Biobank"
+                }
+              ]
+            },
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/translation",
+              "extension": [
+                {
+                  "url": "lang",
+                  "valueCode": "en-US"
+                },
+                {
+                  "url": "content",
+                  "valueString": "Description of the collection/biobank"
+                }
+              ]
+            }
+          ]
+        },
+        "_definition": {
```

Comment:
I get the following error if I load the biobank package with the validator:

```
Load de.medizininformatikinitiative.kerndatensatz.biobank#2025.0.1Exception in thread "main" org.hl7.fhir.exceptions.FHIRException: Error reading /Users/akiel/.fhir/packages/de.medizininformatikinitiative.kerndatensatz.biobank#2025.0.1/package/StructureDefinition-Organization.json from package de.medizininformatikinitiative.kerndatensatz.biobank#2025.0.1: Error loading /Users/akiel/.fhir/packages/de.medizininformatikinitiative.kerndatensatz.biobank#2025.0.1/package/StructureDefinition-Organization.json: Cannot invoke "String.replace(java.lang.CharSequence, java.lang.CharSequence)" because the return value of "org.hl7.fhir.r5.model.ElementDefinition.getDefinition()" is null
```

I think, that not having a definition value can be a problem here.